flink-runtime-web/web-dashboard/app/partials/jobs/job.jade

```diff
@@ -43,6 +43,10 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main(ng-if="job")
     span.navbar-info-button.btn.btn-default(ng-click="cancelJob($event)")
       | Cancel
 
+  .navbar-info.last.first(ng-if!="job.type=='STREAMING' && (job.state=='RUNNING' || job.state=='CREATED')")
```

Comment:
The syntax is quite odd, but it is correct (I figures this out on the Internet). It is necessary to protect `&&` from being changed to HTML `&amp;&amp;` during compilation. I also tested it: STOP button is only displayed for streaming jobs and if I click it the stop shuts down gracefully.